bigframes/session/__init__.py

```diff
@@ -495,14 +494,19 @@ def _get_snapshot_sql_and_primary_key(
 
         job_config = bigquery.QueryJobConfig()
         job_config.labels["bigframes-api"] = api_name
-        current_timestamp = list(
-            self.bqclient.query(
-                "SELECT CURRENT_TIMESTAMP() AS `current_timestamp`",
-                job_config=job_config,
-            ).result()
-        )[0][0]
+        job_config.use_query_cache = use_cache
```

Comment:
Not the `QueryJobConfig` I wanted you to update. Remove, please. This is the query we use to determine the snapshot time, so if anything we'd want to always set this to False.